### PR TITLE
fix: color updates

### PR DESCRIPTION
**Pull request by eboyer**
@LeoT7508 provided some minimal updates to the colors, they are reflected here

### CODE REVIEW

**Comment by joseegm** [comment]
Test Version for this PR was deployed

Built with commit 93dbef314ada22ee399c6975e41beb982d5b391d

https://deploy-preview-1358--fundamental.netlify.com

File: scss/_settings.scss [hunk 1]
```diff
@@ -91,44 +91,50 @@ $fd-colors: map-merge((
...
     12: hsl(0, 60%, 54%), // #D04343 accessible accent 2 B (4.6:1)
     13: hsl(210, 64.2%, 47.1%), // #2B78C5 accessible accent 6 B (4.6:1)
     14: hsl(238, 65.1%, 62.9%), // #0F828F accessible accent 7 B (4.6:1)
-    15: hsl(274.2, 60%, 55.9%), // #647887 accessible accent 10 B (4.6:1)
+    15: hsl(274.2, 60%, 55.9%) // #647887 accessible accent 10 B (4.6:1)
   ),
   "shell": (
-    1: hsl(210, 28%, 29%), // #354A5F shell background
-    2: hsl(210, 100%, 91%) // #D1E8FF shell icons
+    1: hsl(210, 28%, 29%), // #354A5F
```
Comment (droshev) [inline scss/_settings.scss]: those colors are not reflected in https://deploy-preview-1359--fundamental.netlify.com/foundation/colors.html

**Comment by droshev** [comment]
@eboyer maybe you want to update the text colors so it would be readable:
Screen Shot 2019-03-27 at 11 38 26 AM

File: scss/mixins/_breakpoints.scss [hunk 2]
```diff
@@ -0,0 +1,97 @@
+
```
Comment (bcullman) [inline scss/mixins/_breakpoints.scss]: this breakpoint effort seems unrelated to colors.
